client/rest-high-level/src/test/java/org/elasticsearch/client/documentation/ILMDocumentationIT.java

```diff
@@ -201,8 +206,7 @@ public void testGetLifecyclePolicy() throws IOException, InterruptedException {
         ActionListener<GetLifecyclePolicyResponse> listener =
             new ActionListener<GetLifecyclePolicyResponse>() {
                 @Override
-                public void onResponse(GetLifecyclePolicyResponse response)
-                {
+                public void onResponse(GetLifecyclePolicyResponse response) {
```

Comment:
I'm a little surprised this passes the length check, I thought it was over. Cool if it is though!

Reply:
oh, that is my bad. probably happened on accident via rebase